priv/gettext/default.pot

```diff
@@ -53,38 +59,38 @@ msgstr ""
 msgid "Language"
 msgstr ""
 
-#: lib/dpul_collections_web/live/search_live.ex:255
+#: lib/dpul_collections_web/live/search_live.ex:311
 #, elixir-autogen, elixir-format
 msgid "Next"
 msgstr ""
 
+<<<<<<< HEAD
```

Comment:
Looks like just a rebase miss. Maybe just re-running the translation tasks would help?